app/controllers/concerns/verify_sp_attributes_concern.rb

```diff
@@ -27,6 +27,7 @@ def update_verified_attributes
     )
   end
 
+  # maw: come back and look at this some more.
```

Comment:
now that we have backfilled the whole table, we can drop the `last_estimated_consent` `||` logic entirely